cmd/kubernetesDeploy.go

```diff
@@ -225,23 +239,39 @@ func runKubectlDeploy(config kubernetesDeployOptions, command command.ExecRunner
 	}
 
 	if config.CreateDockerRegistrySecret {
-		if len(config.ContainerRegistryUser)+len(config.ContainerRegistryPassword) == 0 {
```

Comment:
A slight discrepancy between helm and kubectl deploy. In the helm deploy method the if looked like so:
`if len(config.ContainerRegistryUser) == 0 || len(config.ContainerRegistryPassword) == 0 {`

I would like to choose one way of checking this. Please give feedback.

Reply:
do not add the lengths:  len(config.ContainerRegistryUser)+len(config.ContainerRegistryPassword)

Reply:
good, I will fix that

Reply:
done